tasks/broad/RNAWithUMIsTasks.wdl

```diff
@@ -98,7 +98,7 @@ task Fastp {
     File adapter_fasta = "gs://gcp-public-data--broad-references/RNA/resources/Illumina_adapters.fasta"
 
     String docker = "us.gcr.io/broad-gotc-prod/fastp:1.0.0-0.20.1-1649253500"
-    Int memory_mb =  "16384"
+    Int memory_mb =  "32000"
```

Comment:
To fix your issue you need to determine this programatically based on the size of the input files + some buffer

We regularly have tasks in skylab and whole genome pipelines that use 100s of gb of memory so its totally fine to scale up

Reply:
Yea I'm running some tests to figure out what this formula should be in our case.